android/lib/pages/profile_page.dart

```diff
@@ -269,6 +281,51 @@ class _ProfilePageState extends State<ProfilePage> {
                           ),
                         ],
                       ),
+                      const SizedBox(height: 10.0),
+                      (profile_username == null ||
+                              current_user.username ==
+                                  user_account_info.username)
```

Comment:
We should format this file by renaming these variables. They should follow the convention. For example this one should have been `userAccountInfo`.

Reply:
Renamed those variables